src/less/variables.less

```diff
@@ -4,16 +4,43 @@
 
 @blue: #759FEB;
 @blue-hover: #759FEB;
+@blue-dark: #5873af;
 
-@red: rgb(209, 72, 54);
-@red-hover: rgb(198, 55, 36);
+@red: #D14836;
+@red-hover: #C63724;
+@red-dark: #A23424;
 
 @green: #0AB51F;
 @green-hover: #0E8D1E;
 @green-dark: #055F05;
 
 @orange: #FFD800;
 @orange-hover: #FFA200;
+@orange-dark: #C77700;
+
+@purple: #cb00ff;
+@purple-hover: #a80091;
+@purple-dark: #880072;
+
+@cyan: #00c7ff;
+@cyan-hover: #009bd0;
+@cyan-dark: #00739c;
+
+@silver: #c0c0c0;
+@silver-hover: #9d9d9d;
+@silver-dark: #7e7e7e;
+
+@bronze: #cd7f32;
+@bronze-hover: #cd7700;
+@bronze-dark: #9c5400;
+
+@orchid: #da70d6;
+@orchid-hover: #b259ae;
+@orchid-dark: #854381;
+
+@pear: #d1e231;
```

Comment:
I admire you for looking for all these values :D Creative man :P

Reply:
The hardest part of this PR XD